esphome/components/ltr303/ltr303.cpp

```diff
@@ -0,0 +1,322 @@
+#include "ltr303.h"
+#include "esphome/core/application.h"
+#include "esphome/core/log.h"
+#include "esphome/core/helpers.h"
+
+namespace esphome {
+namespace ltr303 {
+
+static const char *const TAG = "ltr303";
+static const int ALS_GAIN[8] = {1, 2, 4, 8, 0, 0, 48, 96};                        // multiplier
+static const int ALS_INT_TIME[8] = {100, 50, 200, 400, 150, 250, 300, 350};       // ms
+static const int ALS_MEAS_RATE[8] = {50, 100, 200, 500, 1000, 2000, 2000, 2000};  // ms
```

Comment:
As above, these could save some memory by making them `uint16_t`.
```suggestion
static const uint16_t ALS_INT_TIME[8] = {100, 50, 200, 400, 150, 250, 300, 350};       // ms
static const uint16_t ALS_MEAS_RATE[8] = {50, 100, 200, 500, 1000, 2000, 2000, 2000};  // ms
```